docs/medical-api/getting-started/quickstart.mdx

```diff
@@ -241,7 +242,21 @@ Data Query](/medical-api/api-reference/fhir/consolidated-data-query-post) endpoi
 We also provide the ability to add any FHIR resources to a Patient with our [Create Patient's Consolidated
 Data](/medical-api/api-reference/fhir/create-patient-consolidated) endpoint.
 
-## 9. Share Medical Data with the Networks 🚀🚀🚀
+## 9. Generate Patient Medical Record Summary ⭐⭐⭐
+
+The aforementioned FHIR resources are generated from all available CDAs for a patient, resulting in a set of unique medical records. To condense all of these records into a single document, you can follow these steps:
+
+- kick off a [Consolidated Data Query](/medical-api/api-reference/fhir/consolidated-data-query-post).
```

Comment:
nit: capital K "Kick"

Reply:
also if you wanted, could make section 10 consistent with the rest and capitalize its sentences at the start of the dashed list too

Reply:
Done :)